charts/headlamp/values.schema.json

```diff
@@ -35,7 +35,14 @@
       "type": "array",
       "description": "Image pull secrets",
```

Comment:
Would you mind fixing this description too?

Currently someone looking at the description for ImagePullSecrets is told imagePullSecrets is...  "Image pull secrets"

The description from the documentation is good
https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#containers

> ImagePullSecrets is an optional list of references to secrets in the same namespace to use for pulling any of the images used by this PodSpec. If specified, these secrets will be passed to individual puller implementations for them to use. More info: https://kubernetes.io/docs/concepts/containers/images#specifying-imagepullsecrets-on-a-pod

Reply:
Thank you, will fix it